pkg/apis/eksctl.io/v1alpha5/types.go

```diff
@@ -981,6 +982,8 @@ type (
 		PublicKeyName *string `json:"publicKeyName,omitempty"`
 		// +optional
 		SourceSecurityGroupIDs []string `json:"sourceSecurityGroupIds,omitempty"`
+		// +optional
```

Comment:
Here you could add a link to `introduction#ssh-access` similar to https://github.com/weaveworks/eksctl/blob/9e9e1ee8454fed0f8c2c343b4bb4b15243cad29d/pkg/apis/eksctl.io/v1alpha5/types.go#L567